src/globalComponents/apiCalls/tankAPIIntegration.js

```diff
@@ -337,6 +339,41 @@ function getTanksById(tankIds: Array<string>, onLoad:(tanks: Array<Tank>) => voi
 	)
 }
 
+function updateCasusCode(tankId: string, casusCode: ContainerBlock): void {
```

Comment:
This is duplicate code from `/casus/saveCasus.js`. You should probably make saveCasus pass in an optional tankId and make it only load from a cookie if the passed in id is null. Then we only need to add ~2 lines instead of ~50